models/things.js

```diff
@@ -116,7 +116,7 @@ var Things = {
 
      // Notify each open websocket of the new Thing
      this.websockets.forEach(function(socket) {
-       socket.send(JSON.stringify(thing.getDescription()));
+       socket.send(JSON.stringify(thing));
```

Comment:
I was a bit wrong about the behavior of `Things.getNewThings()`. The Thing descriptions returned by it are actually from `device.asThing`, not true Thing objects.

I'm not sure what the best way to fix this is, but I don't think that adding the `id` key to `getDescription` is correct. Manually adding back the id key also feels less-than-ideal.